### PR TITLE
DA-1939 add more jira and github fields to contributor summary

### PR DESCRIPTION
**Pull request by code-sleuth**
This PR adds:

- Github PRs closed
- Github PRs in Open state
- Github Issues Average number of days an Issue is in Open state for an assignee
- Github Issues Assigned to a contributor
- Jira issues closed by a contributor.

### CODE REVIEW

**Comment by lukaszgryglicki** [comment]
BTW: did you try this somewhere or just coded that blindly? Just wonder... it's a very new code ES/SQL.

**Comment by code-sleuth** [comment]
@lukaszgryglicki I tried it on my machine and its working 😅

**Comment by lukaszgryglicki** [comment]
OK, so please check CSV export too - new fields show also show up.

File: elastic/service.go [hunk 1]
```diff
@@ -464,6 +469,8 @@ func (s *service) additionalWhere(dataSourceType, sortField string) (string, err
 			return `and \"time_to_close_days\" is not null`, nil
 		case "jira_comments":
 			return `and \"comment_id\" is not null and \"type\" = 'comment'`, nil
+		case "jira_issues_closed":
+			return `and \"assignee_uuid\" is not null and \"status\" = 'Closed' or \"status\" = 'Resolved' or \"status\" = 'Done'`, nil
```
Comment (lukaszgryglicki) [inline elastic/service.go:473]: If you use `or` condition - it should be in `(` imho `)` becaus eall those conditions are merged together (`ANDed`) to form a final condition.

Comment (lukaszgryglicki) [reply elastic/service.go:473]: Actually, I would give `and \"status\" in ('Closed', 'Resolved', 'Done')`

File: elastic/service.go [hunk 2]
```diff
@@ -496,6 +506,10 @@ func (s *service) additionalWhere(dataSourceType, sortField string) (string, err
 		switch sortField {
 		case "github_pull_request_prs_created":
 			return `and \"id\" is not null and \"pull_request\" = true`, nil
+		case "github_pull_request_prs_open":
+			return `and \"state\" = 'open'`, nil
```
Comment (lukaszgryglicki) [inline elastic/service.go:510]: I would keed `and \"id\" is not null` part

Comment (code-sleuth) [reply elastic/service.go:510]: on it

File: elastic/service.go (hunk 1 above)
Comment (code-sleuth) [reply elastic/service.go:473]: fixed

**Comment by lukaszgryglicki** [comment]
Actually also one part of `elastic/service.go` is missing
Just search for `jira` (without case) in those 3 files:
- `elastic/service.go` (`JiraIssuesCreated` etc).
- `swagger/dev-analytics-affiliation.yaml (should be fine already).
- `affiliation/service.go` (CSV stuff).

+ rebase.

**Comment by lukaszgryglicki** [comment]
And post-merge /lgtm :P

**Pull request merged**